src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,137 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void containsValidUser_Ok() {
+        User kate = new User();
+        kate.setAge(40);
+        kate.setLogin("1234566666");
+        kate.setPassword("43434343");

```

Comment:
Please move creation of default ok user to BeforeEach method and in tests only change 1 required field(e.g. set login to null, or password to short one, etc) and perform test.
This will significantly reduce amount of code and highlight what actually we test